project.clj

```diff
@@ -36,7 +36,8 @@
                                      :password :env/clojars_jenkins_password
                                      :sign-releases false}]]
 
-  :profiles {:dev {:dependencies [[puppetlabs/http-client]
+  :profiles {:dev {:dependencies [[org.bouncycastle/bcpkix-jdk15on]
```

Comment:
```suggestion
  :profiles {:dev {:dependencies [[org.bouncycastle/bcpkix-jdk18on]
```
We are trying to eliminate the old `15on` in favor of the `18on` which is the newer version.